src-vue/src/views/DeveloperView.vue

```diff
@@ -207,7 +212,22 @@ export default defineComponent({
             await persistentStore.clear();
             // ...and save
             await persistentStore.save();
-        }
+        },
+        async forceInstallNorthstar() {
+            let game_install = {
+                game_path: this.$store.state.game_path,
+                install_type: this.$store.state.install_type
+            } as GameInstall;
+            let install_northstar_result = invoke("install_northstar_caller", { gamePath: game_install.game_path, northstarPackageName: ReleaseCanal.RELEASE });
+            await install_northstar_result
+                .then((message) => {
+                    console.log(message);
```

Comment:
Is this needed?

Reply:
No, removed in 1ffd1c3bb404db3c69b6242b2bc25b9051fd0afb (and replaced with `TODO` comment for https://github.com/R2NorthstarTools/FlightCore/pull/154#issuecomment-1412881556 :P)